src/EmojiTranslatorContainer.js

```diff
@@ -16,10 +24,21 @@ export default class EmojiTranslatorContainer extends Component {
   handleTranslate(event) {
     this.setState({
       originalContent: event.target.value,
-      translatedContent: event.target.value.toUpperCase()
+      //translatedContent: event.target.value.toUpperCase()
+      translatedContent: this.mapOC(event.target.value)
     })
   }
 
+  mapOC(text){
```

Comment:
does this function translate the text? I would give this a better name - `mapOC` is pretty unclear